[solution]: easy-tuple-to-object (#250)
ghaiklor/type-challenges-solutions: en/easy-tuple-to-object.md

those values in `T[number]` and return a new type where the key and value is the
type from `T[number]`:

> We need to make sure that we pass only types contain valid keys. As we know, valid keys in typescript are `string`, `number` or `symbol`. Also, we can you a built-in type `PropertyKey`.

Reviewer: ```suggestion
We need to make sure that we pass only types containing valid keys. As we know, valid keys in TypeScript are `string`, `number` or `symbol`. To achieve that, we can use the built-in type `PropertyKey`.
```
